src/components/hosts/HostListItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Badge } from "reactstrap";
+import { useTranslation } from "react-i18next";
+import { HostContext } from "components/host/HostContext";
+
+const HostStatusItem = (status) => {
+    return <Badge>status</Badge>;
+};
+
+const HostLanguageSpokenItem = (languages) => {
+    return (
+        <div>
+            {languages.map((language, index) => (
+                <Badge>{language}</Badge>
+            ))}
+        </div>
+    );
+};
```

Comment:
Please refactor all such methods as sub-components or at least keep them inside the main component - they will have a dependency on `useTranslation()` later on. Take a look at the approach used for `AccomodationListItem.jsx`

Reply:
Please also provide a context to them in a similar fashion as in the beforementioned example